docs/backend/widgets.md

```diff
@@ -11,6 +11,9 @@ myst:
 
 # Widgets
 
+```{seealso}
+See the chapter {ref}`training:dexterity-reference-label` from the Mastering Plone 6 Training
+```
```

Comment:
This link goes to the top of this page, instead of an anchor in the page that is more relevant:

https://training.plone.org/5/mastering-plone/dexterity_reference.html#widgets

However that only links to the Plone 5.2 docs, which is not a great idea.

https://docs.plone.org/external/plone.app.dexterity/docs/reference/widgets.html

But that brings up another larger question. Should we pull the Dexterity docs into Documentation?

https://github.com/plone/plone.app.dexterity/tree/master/docs

Reply:
The Dexterity-reference from the training (https://training.plone.org/5/mastering-plone/dexterity_reference.html) and the dexterity docs should both move into docs. I would not pull the dexterity-docs in (as a submodule) but rather move it there since it is very very "core".

Reply:
Issue created: https://github.com/plone/documentation/issues/1298

Resolving this comment.